### ISSUE TITLE
Make EPerson "Contact Telephone" field available for Administrators via GUI

### ISSUE THREAD

**Issue by Taurus2024** [DSpace/dspace-angular#2692]
**Is your feature request related to a problem? Please describe.**
Administrator can see and edit data in EPerson "Contact Telephone" field via GUI of Dspace v6.x.
Administrator can not see and edit data in EPerson "Contact Telephone" field via GUI of Dspace v7.x.

**Describe the solution you'd like**
It would be great to allow Administrator to see and edit EPerson "Contact Telephone" field via GUI of Dspace v7.x in the same way as in Dspace v6.x.

**Additional context**
![image](https://github.com/DSpace/dspace-angular/assets/153017429/d7b77b79-40f7-4445-9d78-af07d4ae1b41)
![image](https://github.com/DSpace/dspace-angular/assets/153017429/16eb32c3-6a8e-459c-8241-1150dc519c9f)

**Comment by tdonohue** [comment]
Agreed, this seems reasonable.  Moving this ticket to the 8.0 board in search of a volunteer.  It might not be a difficult task (assuming that the REST API already supports modifying the `Phone` field).

**Comment by tdonohue** [comment]
@Taurus2024 : That was information specific to 7.6 (which was [released on June 23](https://github.com/DSpace/dspace-angular/releases/tag/dspace-7.6) of this year).   If you read that full announcement it notes this:
> Therefore, the DSpace Steering Group would like to announce that the 7.x platform will move into “maintenance mode” immediately after the 7.6 release.  
> * No new features will be added to the 7.x platform after 7.6. Instead, all future releases of 7.x will be bug-fix only releases.

Because 7.6 has been released, this means **no more new features will be added to 7.6.x** regardless of whether the feature was in 6.x.

All that said, if the person who volunteers to tackle this ticket finds that this fix is very minor, at that point we could recategorize it as a "bug fix" and allow it in a 7.6.x release.  But, if they find this is a "new feature" and requires significant improvements to both the backend and frontend, then this will need to stay scheduled for 8.0.

Hopefully that explains things better.  Overall, this is waiting on a volunteer to tackle it, regardless of which release it ends up in.

**Comment by Taurus2024** [comment]
Hi @tdonohue, 
thank you for the detailed feedback. It is clear now.

**Comment by PitbaranK** [comment]
Hi Team,
We'd like to claim #2692.

**Comment by tdonohue** [comment]
@PitbaranK : Assigning to you.  (When you get a chance, could you add your organization, Virsoftech, in your GitHub profile in some manner?  It's really helpful to me when I'm writing future release notes, as it's a reminder to me that these tickets should be credited back to Virsoftech.)

**Comment by Taurus2024** [comment]
Dear DSpace Developer Team,
with full of respect to the Developer Team, I would like to share some contemplates on the topic: if one removes in 2023 one of the key features for the repository that has more than 16K users and there is no solution 3 years later, then value of the feature for this repository could be approaching to zero.
Nevertheless, I am confident that the feature could be very useful for recently created or small repositories.

**Comment by tdonohue** [comment]
@Taurus2024 : Our developer team is entirely volunteer based...we do not have centralized group of developers or code reviewers/testers.  This feature has been delayed because it wasn't claimed (by a volunteer developer) until Dec 2024, at which point a PR was created in #3740.  That PR underwent initial feedback in early 2025, but the volunteer developer didn't get back to it until late 2025.  It's waiting now for an updated test/review from a volunteer in our community to help us move the feature forward.

Therefore, if you are dependent on this feature, we'd welcome you to test or review PR #3740 to verify that the solution there works properly for you.  Once PRs are verified/tested, we move them into a "Reviewer Approved" column of our [active project board](https://github.com/orgs/DSpace/projects/35/views/1), where we will attempt to get them merged into an upcoming release.

**Comment by Taurus2024** [comment]
Dear @tdonohue ,
Thank you for your prompt and detailed reply.
I see what you're saying about the lack of human resources, and I completely understand it.
From my perspective, DSpace has become a key (or even core) system for many organizations of different sizes around the world. This means that even a small bug or a change in functionality can have a significant impact on organizations, their processes, and, in some cases, their finances.
With that in mind, perhaps it's worth considering whether some development or release processes could be adjusted a bit to help reduce the risks for organizations that rely on DSpace as a core system. It's just as an idea.
In any case, I really appreciate all the volunteers and the hard work they put into this great project.

Unfortunately, I am not either developer or tester. But I will try to test the suggested solution once I understand what exactly I have to do to implement it :)
I am guessing that to implement the solution I have to update files by adding green lines as it is on the picture, right?

<img width="1196" height="732" alt="Image" src="https://github.com/user-attachments/assets/42564d4a-a1d2-4fa5-adbe-54d6db191ecd" />

**Comment by tdonohue** [comment]
@Taurus2024 : Yes, essentially, you'd have to add the changes in that PR to your code.  The green changes are things to add, and any red changes are things to remove. 

However, there are easier ways to do this for someone who has knowledge of Git/GitHub.  So, if you have a developer available, you may ask them to do this on your behalf.  We also do have some (in progress) instructions for how to use Docker to test PRs here: https://wiki.lyrasis.org/spaces/DSPACE/pages/345079816/Testing+DSpace+Github+Pull+Requests

Another option altogether would be to hire a [Service Provider](https://dspace.org/registered-service-providers/) to help you get this feature into DSpace core code.  For institutions that do not have developers on staff, hiring a service provider is a way to more directly get something added to DSpace on your behalf.

Ultimately, changes only get added to DSpace if someone can test/approve them.  Since this one has had very little attention, that tells me that most DSpace sites may not use this feature.  Often, bug fixes to widely used features will move along more quickly & be released rapidly because volunteers will be found quickly via institutions that rely on the feature (or have paid a service provider to claim the task and fix the bug).
